embed-gutenberg-block-google-maps.php

```diff
@@ -14,168 +14,171 @@
-    
-    // Check status code of apiURL
-    $ch = curl_init($apiURL);
-    curl_setopt($ch, CURLOPT_HEADER, true);
-    curl_setopt($ch, CURLOPT_NOBODY, true);
-    curl_setopt($ch, CURLOPT_RETURNTRANSFER,1);
-    curl_setopt($ch, CURLOPT_TIMEOUT,10);
-    $output = curl_exec($ch);
-    $httpcode = curl_getinfo($ch, CURLINFO_HTTP_CODE);
-    curl_close($ch);
-
-    // Don't output anything if the response from Google Maps isn't a 200
-    if( $httpcode !== 200 ){
-        return;
-    }
-
-    // Set the appropriate CSS class names
-    $classNames = ( $interactive ) ? "wp-block-pantheon-google-map interactive ratio$aspectRatio" : "wp-block-pantheon-google-map";
-    
-    // Create the output
-    $output = "<div class='$classNames'><div class='map'>";
-    // If the map is interactive show the iframe
-    if( $interactive ){
-        $output .= "<iframe width='100%' height='100%' frameborder='0' style='border:0' src='$apiURL' allowfullscreen></iframe>";
-    // Otherwise use the static API
-    } else {
-        $output .= "<img src='$apiURL' />";
-    }
-    $output .= '</div></div>';
-
-    // Return the output
-    return $output;
+function render_gutenberg_map_embed_block( $attributes ) {
+
+	// Get the API key.
+	$api_key = get_option( 'pantheon_google_map_block_api_key' );
+
+	// Don't output anything if there is no API key.
+	if ( null === $api_key || empty( $api_key ) ) {
+		return;
+	}
+
+	// Expand all the attributes into separate variables.
+	foreach ( $attributes as $key => $value ) {
+		${$key} = $value;
+	}
+
+	// URL encode the location for Google Maps.
+	$location = rawurlencode( $location );
+
+	// Set the API URL based to embed or static maps based on the interactive setting.
+	$api_url = ( $interactive ) ? "https://www.google.com/maps/embed/v1/place?key=${api_key}&q=${location}&zoom=${zoom}&maptype=${mapType}" : "https://maps.googleapis.com/maps/api/staticmap?center=${location}&zoom=${zoom}&size=${maxWidth}x${maxHeight}&maptype=${mapType}&key=${api_key}"; /* phpcs:ignore */
+
+	// Check status code of the API.
+	// phpcs:disable WordPress.WP.AlternativeFunctions
+	$ch = curl_init( $api_url );
+	curl_setopt( $ch, CURLOPT_HEADER, true );
+	curl_setopt( $ch, CURLOPT_NOBODY, true );
+	curl_setopt( $ch, CURLOPT_RETURNTRANSFER, 1 );
+	curl_setopt( $ch, CURLOPT_TIMEOUT, 10 );
+	$output   = curl_exec( $ch );
+	$httpcode = curl_getinfo( $ch, CURLINFO_HTTP_CODE );
+	curl_close( $ch );
+	// phpcs:enable WordPress.WP.AlternativeFunctions
+
+	// Don't output anything if the response from Google Maps isn't a 200.
+	if ( 200 !== $httpcode ) {
+		return;
+	}
+
+	// Set the appropriate CSS class names.
+	$classes = ( $interactive ) ? "wp-block-pantheon-google-map interactive ratio$aspectRatio" : 'wp-block-pantheon-google-map'; /* phpcs:ignore */
```

Comment:
`/* phpcs:ignore */` ignores two lines, would be good to put it on line 117.5

Non-blocker